backends/bmv2/psa_switch/psaSwitch.cpp

```diff
@@ -380,6 +395,12 @@ bool InspectPsaProgram::preorder(const IR::Parameter* param) {
     if (!ft->is<IR::Type_StructLike>())
         return false;
     auto st = ft->to<IR::Type_StructLike>();
+    // check if it is psa specific standard metadata
+    if (isStandardMetadata(st)) {
+      addHeaderType(st);
+      // remove struct and _t from type name
+      addHeaderInstance(st, (st->toString()).substr(7, st->toString().size()-9));
```

Comment:
Can we write this in a more clean way.

Reply:
I changed it so we're using the type name instead of the struct name so we just have to deal with getting rid of the _t at the end which cstring has a handy function for. Let me know if you wanted something more.

Reply:
Looks good!